student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(classroom, age, name = 'Unknown', parent_permission: true)
   def play_hooky
     "¯\(ツ)/¯"
   end
+
+  def student=(student)
+    @student = student
+    student.students.push(self) unless student.students.include?(self)
+  end
```

Comment:
- [ ]  there should be a method that serves as the 'belongs-to' side (a student belongs to a classroom). have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) with examples. this might look something like below: 
```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```
- [ ] and there should be an attribute reader for `classroom` as well